examples/cdi-produces-disposes/README_pt.adoc

```diff
@@ -0,0 +1,291 @@
+= CDI Produces Disposes
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+Esté exemplo demostra como as anotaçõẽs @Produces e @Disposes trabalham. Um LogFactory cria uma instância do LogHandler
+dependendo de um atributo "type". Para o propósito deste exemplo, o type é hard-coded para especificar um valor.
+Uma implementação Logger deve contér uma lista de LogHandlers. Teremos três implementações da interface LogHandler.
+
+* Uma DatabaseHandler
+* Uma FileHandler
+* Uma ConsoleHandler
+
+O DatabaseHandler aparentemente gravar os logs em um banco de dados. O FileHandler aparentemente gravar os logs em um arquivo.
+O ConsoleHandler seria apenas imprimir os logs para um console (saída padrão). Esté exemplo é apenas uma ilustração de como
+os conceitos dentro do CDI trabalha e é não destina-se a fornecer uma implementação/design de um framework de log.
```

Comment:
"trabalha e é não"

Reply:
What is wrong here?

Reply:
O que "trabalha" é o "conceito", então nesse caso como "conceito" esta no plural, seria "os conceitos dentro do CDI trabalham", mas nesse caso work pode ser traduzido para "como os conceitos dentro do CDI funcionam". E nessa parte "e é não" seria apenas retirar o "é".

Reply:
very good, thanks :  )

Reply:
done.